Corona-Warn-App/src/main/res/layout/home_statistics_cards_keysubmissions_layout.xml

```diff
@@ -1,24 +1,150 @@
+            app:flow_wrapMode="chain"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_chainStyle="packed"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/primaryLabel"
+            style="@style/StatisticsCardValueLabel"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:text="@{keySubmissionsLabel}"
+            tools:text="Gestern" />
+
+        <TextView
+            android:id="@+id/primaryValue"
+            style="@style/StatisticsCardPrimaryValue"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:includeFontPadding="false"
+            android:text="@{keySubmissions}"
+            tools:text="1.514" />
+
+        <TextView
+            android:id="@+id/secondaryLabel"
+            style="@style/StatisticsCardValueLabel"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:paddingTop="@dimen/spacing_tiny"
+            android:text="@string/statistics_card_infections_secondary_label" />
+
+        <TextView
+            android:id="@+id/secondaryValue"
+            style="@style/StatisticsCardSecondaryValue"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@{sevenDayAverage}"
+            tools:text="1.812" />
+
+        <de.rki.coronawarnapp.statistics.ui.TrendArrowView
+            android:id="@+id/trendArrowView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/spacing_tiny"
+            app:layout_constraintBottom_toBottomOf="@id/secondaryValue"
+            app:layout_constraintStart_toEndOf="@id/secondaryValue"
+            app:layout_constraintTop_toTopOf="@id/secondaryValue" />
+
+        <TextView
+            android:id="@+id/tertiaryLabel"
+            style="@style/StatisticsCardValueLabel"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:paddingTop="@dimen/spacing_tiny"
+            android:text="@string/statistics_card_infections_tertiary_label" />
+
+        <TextView
+            android:id="@+id/tertiaryValue"
+            style="@style/StatisticsCardSecondaryValue"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@{total}"
+            tools:text="20.992" />
+
+        <TextView
+            style="@style/StatisticsCardValueLabel"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/card_padding"
+            android:layout_marginBottom="@dimen/spacing_tiny"
```

Comment:
That margin seems a little small - should be 16 dp according to figma == spacing_small